### DISCUSSION TITLE
Give users instructions on how to deactivate AirTags

### DISCUSSION THREAD

**Discussion by SteadyStatus21** [seemoo-lab/AirGuard#115] [Ideas]
Hello. I believe that you should add a tutorial on how to remove the battery from a airtag. This would be beneficial to those who do not understand how to do so.

If you cannot do that, try linking to a article that will explain how to deactivate the airtag.

Thanks,
SteadyStatus21

**Comment by lbdroid** [comment]
Doesn't everybody know? You smash it with a hammer!
